samples/event-sourcing-esdb-simple/src/main/java/io/eventdriven/ecommerce/shoppingcarts/Events.java

```diff
@@ -0,0 +1,49 @@
+package io.eventdriven.ecommerce.shoppingcarts;
+
+import io.eventdriven.ecommerce.shoppingcarts.productitems.PricedProductItem;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+public final class Events {
+
+  public sealed interface ShoppingCartEvent permits
```

Comment:
zamiast permits lepiej zawrzeć eventy w sealed trait: https://github.com/softwaremill/reactive-event-sourcing-java/blob/main/src/main/java/workshop/cinema/reservation/domain/ShowEvent.java

Reply:
To jest zdecydowanie lepsze podejście, dzięki! Poprawione w https://github.com/oskardudycz/EventSourcing.JVM/pull/1/commits/69572137731b96af9c9bcaa1ef90e0baeb05bf30.